packages/kbn-telemetry-tools/src/tools/tasks/check_matching_schemas_task.ts

```diff
@@ -23,7 +23,7 @@ export function checkMatchingSchemasTask({ roots }: TaskContext, throwOnDiff: bo
         root.esMappingDiffs = Object.keys(differences);
         if (root.esMappingDiffs.length && throwOnDiff) {
           throw Error(
-            `The following changes must be persisted in ${fullPath} file. Use '--fix' to update.\n${JSON.stringify(
+            `The following changes must be persisted in ${fullPath} file. Run 'node scripts/telemetry_check --fix' to update.\n${JSON.stringify(
```

Comment:
Not super happy with coupling `scripts/telemetry_check` in this _separate_ package. On the flip side, this is a mono-repo package, so I guess the harm is small enough vs. the benefit. What do you think?

Reply:
I think the benefit outweighs the concern. Kibana is a huge project with so many checks and scripts that no one developer knows all of them. The only way to manage this is to have explicit error messages that help guide developers to fixing their own CI failures.